.github/workflows/whitespace_checks.yml

```
@@ -32,14 +34,85 @@ jobs:
       - uses: actions/checkout@v4
       # Line endings are already ensured by .gitattributes
       - name: Check trailing whitespaces
-        run: if git ls-files | grep -E '\.txt$|\.md$|\.[ch]$|\.cpp$|\.hpp$|\.sh$|\.cmake$|\.glsl$' | xargs grep -n '\s$'; then echo -e "\033[0;31mFound trailing whitespace"; (exit 1); else (exit 0); fi
+        run: |
+          if git ls-files |\
+            grep -E '\.txt$|\.md$|\.[ch]$|\.cpp$|\.hpp$|\.sh$|\.cmake$|\.glsl$' |\
+            xargs grep -n '\s$';\
+          then\
+            echo -e "\033[0;31mFound trailing whitespace";\
+            (exit 1);\
+          else\
+            (exit 0);\
+          fi
+
+  indent_spaces:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # Line endings are already ensured by .gitattributes
+      # Multiple multline comments in one line is not supported by this check 
+      # and there is no reason to use them
+      # So lines like: "/* */ /*" or "*/ a = 5; /*" will result in error
+      - name: Check for unsupported multiline comments
+        run: |
+          if git ls-files |\
+            grep -E '^src/.*\.cpp$|^src/.*\.[ch]$' |\
+            xargs grep -n '\*/.*/\*';\
+          then
+            echo -e "\033[0;31mUnsupported combination of multiline comments. New multiline comment should begin on new line.";\
+            (exit 1);\
+          else\
+            (exit 0);\
+          fi
+          if git ls-files |\
+            grep -E '\.lua$' |\
+            xargs grep -n -e '--\]\].*--\[\[';\
```

Also not `--]]`. It's just `]]`.

```suggestion
            xargs grep -n -e '\]\].*--\[\[';\
```

(If you see `--]]` being used somewhere, that's because someone wants the end to become a single line comment when they remove the starter. FYI.)

---

I forgot about this totally. :)